src/Navbar.tsx

```diff
@@ -294,7 +295,7 @@ class Navbar extends React.Component<any, State> {
                         <Dropdown className={this.getBorder(topLevelTo)}>
                           <Dropdown.Toggle
                             variant="light"
-                            id={displayName}
+                            id={`Navbar-dropdown-${displayName}`}
```

Comment:
This was "Organizations" in one portal, which matched a title.  The original solution (that used react-router-hash-link) relied on the element being scrolled to having an ID that matched the hash value (just like standard html).  That's no longer the case, but I left in this change.